test/TestServerProjectsLowLevel/body-complex/Generated/PolymorphismClient.cs

```diff
@@ -59,18 +59,111 @@ public PolymorphismClient(AzureKeyCredential credential, Uri endpoint, AutoRestC
         /// <summary> Get complex types that are polymorphic. </summary>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <remarks>
-        /// Schema for <c>Response Body</c>:
+        /// Below is the JSON schema for the request and response payloads.
```

Comment:
It looks like the HTML to handle polymorphic types may not have been implemented yet.  Please see the [issue description](https://github.com/Azure/autorest.csharp/issues/2288) for this feature, and [Shivangi's reference implementation](http://raw.githack.com/ShivangiReja/azure-sdk-for-net/SampleAPIDocs/doc/ApiDocGeneration/Azure.Analytics.Purview.Scanning/_site/api/Azure.Analytics.Purview.Scanning/Azure.Analytics.Purview.Scanning.PurviewScanClient.html#Azure_Analytics_Purview_Scanning_PurviewScanClient_CreateOrUpdate_Azure_Core_RequestContent_Azure_RequestContext_) for specific details.